test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -372,129 +409,6 @@
       "./dev-resources/config/jetty/ssl/certs/master-with-intermediate-ca.pem"
       "./dev-resources/config/jetty/ssl/certs/ca-root.pem")))
 
-(deftest determine-max-threads-test
```

Comment:
These are all gone now because they were only needed to validate the auto-adjustment of `max-threads` from TK-130, which would no longer exist per this PR.